docs/packages/node/README.md

```diff
@@ -232,18 +232,21 @@ module.exports = {
         async: true
       },
 
+      // Target specific versions via babel-preset-env
+      targets: {
+        node: '6.10'
+      }
+
       // Add additional Babel plugins, presets, or env options
       babel: {
         // Override options for babel-preset-env, showing defaults:
         presetEnv: {
```

Comment:
Hmm, there is no more `presetEnv` here, must be a typo. Can you update this to:

```js
babel: {
  presets: [
    ['babel-preset-env', {
```

?

Reply:
Ah yep - I wondered about that.